osu.Game/Users/UserPanel.cs

```diff
@@ -217,33 +101,110 @@ protected override void LoadComplete()
             Status.TriggerChange();
         }
 
-        private void displayStatus(UserStatus status, UserActivity activity = null)
+        protected override bool OnHover(HoverEvent e)
+        {
+            BorderThickness = 2;
+            return base.OnHover(e);
+        }
+
+        protected override void OnHoverLost(HoverLostEvent e)
+        {
+            BorderThickness = 0;
+            base.OnHoverLost(e);
+        }
+
+        [NotNull]
+        protected abstract Drawable CreateLayout();
+
+        protected UpdateableAvatar CreateAvatar() => new UpdateableAvatar
+        {
+            User = User,
+            OpenOnClick = { Value = false }
+        };
+
+        protected UpdateableFlag CreateFlag() => new UpdateableFlag(User.Country)
+        {
+            Size = new Vector2(39, 26)
+        };
+
+        protected OsuSpriteText CreateUsername() => new OsuSpriteText
         {
-            const float transition_duration = 500;
+            Font = OsuFont.GetFont(size: 20, weight: FontWeight.Bold, italics: true),
```

Comment:
```suggestion
            Font = OsuFont.GetFont(size: 16, weight: FontWeight.Bold),
```

seems like the new design doesn't use italics anymore